src/bootstrap.config.js

```diff
@@ -0,0 +1,85 @@
+import path from 'path';
+
+import fileExists from './utils/fileExists';
+import parseConfig from './utils/parseConfig';
+import selectModules from './utils/selectModules';
+import selectUserModules from './utils/selectUserModules';
+import getEnvProp from './utils/getEnvProp';
+
+/* ======= Fetching config */
+
+const DEFAULT_VERSION = 3;
+const SUPPORTED_VERSIONS = [3, 4];
+const CONFIG_FILE = '.bootstraprc';
+
+const userConfigPath = path.resolve(__dirname, `../../../${CONFIG_FILE}`);
+const isUserConfig = fileExists(userConfigPath);
```

Comment:
I assume `bootstrap-loader` will be installed in `node_modules`, which is located in root directory of project. And `.bootstraprc` will  be on the same level with `node_modules`.